pkg/provider/azure_loadbalancer.go

```diff
@@ -2225,7 +2233,7 @@ func (az *Cloud) buildHealthProbeRulesForPort(serviceManifest *v1.Service, port
 
 	// if numberOfProbes is not set, set it to default instead ref: https://docs.microsoft.com/en-us/rest/api/load-balancer/load-balancers/create-or-update#probe
 	if numberOfProbes == nil {
-		numberOfProbes = pointer.Int32(consts.HealthProbeDefaultNumOfProbe)
+		numberOfProbes = pointer.Int32(1)
```

Comment:
I don't think Azure's API will allow this will it? The minimum is still 2, they just have a bug and are not observing it IIRC

Reply:
Yeah it's not clear for me TBH, them not observing it should move the count down to 1. So I figured 1 might work.
But I can revert this.